src/cores/stickers/StickerForm.js

```diff
@@ -0,0 +1,229 @@
+import { useState, useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { updateSticker } from '../../features/stickers/stickerSlice'
+import { setAllEvents } from '../../features/events/eventSlice'
+import { useNavigate, useParams, Link } from 'react-router-dom'
+import { saveSticker, editSticker } from '../../services/stickers.services';
+import { fetchAllEvents } from '../../services/events.services';
+import Select from '../../components/select';
+import {fetchAllTeams} from '../../services/team.services'
+import {fetchTeams} from '../../features/teams/teamSlice'
+function StickerForm() {
+
+    const events = useSelector(state => state.events.eventsAll);
+    const eventsOptions = events.map((event) => ({
+        id: event.id,
+        name: event.eventName,
+    }));
+    
+    const teams = useSelector(state => state.teams.teams);
+    const teamsOptions = teams.map((team) =>({
+        id:team.id,
+        name:team.name,
+    }));
```

Comment:
```suggestion
    const teamsOptions = useTeamsOptions();
```
Lo mismo que con events, un hook personalizado, AUNQUE en este caso... "events" dice "eventsAll" asumo son todos los eventos

Pero "teams" realmente son TODOS los teams 👀 ??, se me hace a que estan sacandolo del lugar de la store que deberia PAGINAR teams, no de donde se almacenen todos los teams, sugeriria hacer un "teamsAll".